src/Bicep.Core/Analyzers/Linter/Rules/InterpolateNotConcatRule.cs

```diff
@@ -56,14 +56,13 @@ public override void VisitFunctionCallSyntax(FunctionCallSyntax syntax)
                     var resultType = this.model.GetTypeInfo(syntax);
                     if (resultType is not AnyType && TypeValidator.AreTypesAssignable(resultType, LanguageConstants.String))
                     {
+                        // must have more than 1 argument to use interpolation
+                        if (syntax.Arguments.Length > 1 && CreateFix(syntax) is CodeFix fix)
```

Comment:
nit: do this check first to avoid the more expensive type info calculations if not needed.